package/yast2-caasp.spec

```diff
@@ -0,0 +1,80 @@
+#
+# spec file for package yast2-registration
+#
+# Copyright (c) 2017 SUSE LINUX Products GmbH, Nuernberg, Germany.
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via http://bugs.opensuse.org/
+#
+
+
+Name:           yast2-caasp
+Version:        1.0.0
+Release:        0
+
+BuildRoot:      %{_tmppath}/%{name}-%{version}-build
+Source0:        %{name}-%{version}.tar.bz2
+
+Group:          System/YaST
+License:        GPL-2.0
+
+# SystemRoleHandlersRunner
+Requires:       yast2-installation >= 3.1.217.30
+Requires:       yast2
+
+# SystemRoleHandlersRunner
+BuildRequires:  yast2-installation >= 3.1.217.30
+BuildRequires:  yast2
+BuildRequires:  yast2-devtools >= 3.1.39
+BuildRequires:  rubygem(yast-rake) >= 0.2.13
+BuildRequires:  rubygem(rspec)
+
+BuildArch:      noarch
+
+# FIXME: it seems can we cannot move it to macros.yast, the yast-rake-ci is not
```

Comment:
This is just an experimental code for building pull requests in Jenkins. As we have switched to Docker based Travis builds we do not need this anymore. Please remove this.